docs/source/pages/digitalSignature/listingPage/digitalMassSigningOfEnvelopes.rst

```diff
@@ -0,0 +1,22 @@
+==================================
+Digital mass signing of envelopes
+==================================
+
+.. contents:: Сontent:
+   :depth: 6
+
+---------
+
+You can read how you can add a signature to the document in the previous section.
+
+How to sign several envelopes at once
+======================================
+
+To mass sign the envelopes you have to do the following:
+
+1. Press button "Mailbox" on the left sidebar;
+2. Choose envelopes which you want to sign. You can select only completed envelopes;
+3. Find icon "key". Hover your mouse over the icon and you will see tooltip "Sign envelopes with EDS". Press this button.
+
+As soon as you click on it, the modal window opens. And what is next to do in the modal window, you can read in the topic
```

Comment:
Change text "And what is next to do in the modal window, you can read in the topic" to "You're able to read the key, how to do this you can read in the topic"